modulefiles/build_hera_intel.lua

```diff
@@ -24,6 +24,8 @@ load(pathJoin("hpc-impi", os.getenv("hpc_impi_ver") or "2022.1.2"))
 
 load("srw_common")
 
+unload("netcdf")
+load(pathJoin("netcdf", os.getenv("netcdf_ver") or "4.7.4"))
```

Comment:
I have reached out to Natalie and she has corrected the issue with netcdf/4.9.1 and hdf5/1.14.0 being treated as the defaults on Hera.  Please remove these two lines and make sure that the online-cmaq builds without issue.  Thanks!

Reply:
@MichaelLueken, yes, I had an issue with netcdf on hera. Thank you for the update. I'll remove the lines and test it again.

Reply:
Removed. It works!